.github/workflows/pull_requests.yaml

```diff
@@ -162,3 +162,13 @@ jobs:
             --skip-missing-values \
             --excluded-charts console \
             --target-branch ${{ github.event.repository.default_branch }}
+  summary:
+    if: always()
+    needs:
+      - test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Summarize the results of the test matrix pass/fail
+        uses: re-actors/alls-green@release/v1
+        with:
+          jobs: ${{ toJSON(needs)}}
```

Comment:
```suggestion
        with:
          allowed-failures: check-values
          jobs: ${{ toJSON(needs)}}
```

Reply:
again, this is only for the matrix tests.